maintenance/wikia/updateIpblocksSchema.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+require_once __DIR__ . '/../Maintenance.php';
+
+/**
+ * Maintenance script to delete duplicate blocks from ipblocks table
+ * and update table schema to enforce a stricter constraint
+ *
+ * @see https://wikia-inc.atlassian.net/browse/SUS-1563
+ */
+class UpdateIpblocksSchema extends Maintenance {
+
+	public function __construct() {
+		parent::__construct();
+		$this->addOption( 'really-do-changes', 'Actually delete duplicate rows and update schema.' );
+	}
+
+	public function execute() {
+		list( $allIds, $allIdsCount ) = $this->getDuplicateBlocks();
+
+		if ( $this->hasOption( 'really-do-changes' ) ) {
+			if ( $allIdsCount > 0 ) {
+				$this->deleteDuplicateBlocks( $allIds, $allIdsCount );
+			}
+
+			$this->updateTableSchema();
+		}
+	}
+
+	/**
+	 * Load and print block IDs to be deleted
+	 * @return array list of block IDs to be deleted and count of block IDs to be deleted
+	 */
+	private function getDuplicateBlocks(): array {
+		/*
+			SELECT GROUP_CONCAT(ipb_id) FROM ipblocks GROUP BY ipb_address, ipb_user HAVING count(*) > 1;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS125)